### PR TITLE
clean up & wire up for ui fixes

### PR DESCRIPTION
**Pull request by mattkrick**

### CODE REVIEW

**Comment by mattkrick** [comment]
@ackernaut here are my changes, feel free to selectively merge

File: src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js [hunk 1]
```diff
@@ -100,22 +101,14 @@ class CheckInQuestion extends Component {
...
-    const whichTooltipProps = (isEditing || !isFacilitating) ? tooltipHiddenProps : tooltipProps;
-
     return (
-      <Tooltip {...whichTooltipProps}>
+      <Tooltip
+        tip={<div>{tip}</div>}
+        originAnchor={{vertical: 'bottom', horizontal: 'center'}}
+        targetAnchor={{vertical: 'top', horizontal: 'center'}}
+        hideOnFocus
+        isOpen={isFacilitating && !isEditing}
```
Comment (ackernaut) [inline src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js]: This means that the tooltip will always be open when facilitating?

Comment (ackernaut) [reply src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js]: My hacky looking `whichTooltipProps` meant to enable on hover, except in the case that the facilitator is editing or the user is not the facilitator, where it forces it to be closed.

**Comment by ackernaut** [comment]
@mattkrick thanks! I’ll get this in as soon as I pick back up on the UI updates. I did have a question about the tooltip, but no rush!

File: src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js (hunk 1 above)
Comment (mattkrick) [reply src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js]: ah shoot yeah i need a ternary, forgot the whole default logic

**Comment by mattkrick** [comment]
found a nasty bug that i fixed in `click-card-links` so we'll merge that to epic8.5 & then this will look a lot cleaner

**Comment by ackernaut** [comment]
Thanks for the help here @mattkrick — going to merge this into the UI updates branch in order to push forward.

**Pull request merged**